Documentation/data-type-and-flow-control.md

```diff
@@ -78,13 +78,13 @@ Typestrings = (string) Typestring
 Typestring = (string) { float32, float64, int64, uint64, int32, uint32, int16, uint16, int8, uint8 }
 Dimensions = (string) Dimension
            | (string) Dimension, Dimensions
-Dimension = (string) [1-65535]:[1-65535]:[1-65535]:[1-65535]
+Dimension = (string) [[1-65535]:]{15}[1-65535]
 ```
 
 Note that dimension related values are only effective with static or sparse formats.
 With ```format=flexible```, types and dimensions are not effective.
 
-The buffer of ```other/tensors``` streams have multiple memory chunks. Each memory chunk represents one single tensor in the buffer format of ```other/tensor```. With default configurations of Gstreamer 1.0, the maximum allowed number of memory chunks in a buffer is **16**. Thus, with such configurations of Gstreamer 1.0, ```other/tensors``` may include up to **16** ```other/tensor```.
+The buffer of ```other/tensors``` streams have multiple memory chunks. Each memory chunk represents one single tensor in the buffer format of ```other/tensor```. With default configurations of Gstreamer 1.0, the maximum allowed number of memory chunks in a buffer is **16**. Thus, with such configurations of Gstreamer 1.0, ```other/tensors``` may include up to **16** static ```other/tensor``` and **240** extra ```other/tensor```.
```

Comment:
```suggestion
The buffer of ```other/tensors``` streams has multiple memory chunks. Each memory chunk represents one single tensor in the buffer format of ```other/tensor```. With the default configuration of GStreamer 1.0, a GstBuffer can hold **16** memory chunks at most. Thus,  ```other/tensors``` may include up to **16** static ```other/tensor``` and **240** extra ```other/tensor```.
```

Reply:
Updated!
Thank you 👍